pkg/api/v1/router_server_attributes_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"go.hollow.sh/serverservice/internal/dbtools"
-	hollow "go.hollow.sh/serverservice/pkg/api/v1"
+	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
```

Comment:
is this override necessary given this is the package name now?

Reply:
`gofmt` adds it back if I remove it...so I guess so, at least for now.